.drone.yml

```
@@ -71,3 +71,22 @@ steps:
   - black --config black.toml --check ./rdflib || true
   - flake8 --exit-zero rdflib
   - ./with-fuseki.sh pytest -ra
+
+---
+kind: pipeline
+name: python-3-10
+type: docker
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: test
+  image: python:3.10
+  commands:
+  - pip install --default-timeout 60 -r requirements.txt
```

The failure is because there is no JDK for fuseki to run with:

```suggestion
  - apt-get update && apt-get install -y openjdk-11-jdk-headless
  - pip install --default-timeout 60 -r requirements.txt
```